devices.js

```diff
@@ -5553,12 +5553,12 @@ const devices = [
         vendor: 'ROBB',
         description: 'Zigbee curtain motor controller',
         supports: 'open, close, stop, position',
-        meta: {configureKey: 1},
+        meta: {configureKey: 2},
```

Comment:
To fix https://github.com/Koenkk/zigbee-herdsman-converters/issues/1467 try changing this line to

```
meta: {configureKey: 2, coverInverted: true},
````

Reply:
Thanks. That doesn't seem to do anything. Even after a re-pair it doesn't invert the reported state.

Reply:
It seems to magically work now. What an adventure.